macrosynergy/management/utils/core.py

```diff
@@ -322,3 +321,25 @@ def rec_search_dict(d: dict, key: str, match_substring: bool = False, match_type
                 return result
 
     return None
+
+
+class Timer(object):
```

Comment:
One of us need to write a unit-test for this one. Should be fairly straight forward. Any preferences/